verilog/parser/verilog.y

```diff
@@ -5431,11 +5431,20 @@ loop_generate_construct
 
 conditional_generate_construct
   : generate_if generate_item TK_else generate_item
-    { $$ = MakeTaggedNode(N::kConditionalGenerateConstruct, $1, $2, $3, $4); }
+    { $$ = MakeTaggedNode(N::kConditionalGenerateConstruct,
+             MakeTaggedNode(N::kGenerateIfClause,
+               MakeTaggedNode(N::kGenerateIfHeader, $1),
+               MakeTaggedNode(N::kGenerateIfBody, $2)),
+           MakeTaggedNode(N::kGenerateElseClause,
+               $3,
+               MakeTaggedNode(N::kGenerateElseBody, $4))); }
   | generate_if generate_item %prec less_than_TK_else
-    { $$ = MakeTaggedNode(N::kConditionalGenerateConstruct, $1, $2); }
+    { $$ = MakeTaggedNode(N::kConditionalGenerateConstruct,
+             MakeTaggedNode(N::kGenerateIfClause,
+               MakeTaggedNode(N::kGenerateIfHeader, $1),
+               MakeTaggedNode(N::kGenerateIfBody, $2))); }
```

Comment:
add a nullptr for where the else-clause would go (for structural consistency)

Reply:
Shouldn't we do such thing with `kConditionalStatement` https://github.com/google/verible/blob/master/verilog/parser/verilog.y#L6525?

Reply:
Yes, in general, for all uses of the same `kNodeEnum`.  Some of the `MakeTaggedNode` calls get wrapped into specific functions that expect a fixed number of positional arguments to help with consistency.